src/Views/ArticleView.jsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react"
+import { useParams } from "react-router-dom"
+import { getArticle } from "../__utils__/api"
+
+export default function ArticleView() {
+    const { id } = useParams()
+    const [article, setArticle] = useState({})
+    const [isLoading, setIsLoading] = useState(false)
+
+    useEffect(() => {
+        setIsLoading(true)
```

Comment:
you're setting loading to be true here but not actually doing anything with it. Would be nice to return some html if loading is true 👍